cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -35,7 +35,10 @@
 import org.apache.kafka.common.Node;
 import org.apache.kafka.common.PartitionInfo;
 import org.apache.kafka.common.TopicPartition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import static com.linkedin.kafka.cruisecontrol.model.ModelUtils.*;
```

Comment:
Nit: Can we avoid wildcard import? -- e.g. maybe we can directly use `ModelUtils.hasSameReplicasFor(...)` in the code w/o an import.